packages/preset-create-react-app/helpers/processCraConfig.js

```diff
@@ -0,0 +1,33 @@
+const processCraConfig = (craWebpackConfig, storybookConfigDir) =>
+  craWebpackConfig.module.rules.reduce((rules, rule) => {
+    const { test, oneOf, include } = rule;
+
+    if (test && test.test('.jsx')) {
+      const newRule = {
+        ...rule,
+        include: [include, storybookConfigDir],
+      };
+      return [...rules, newRule];
+    }
+
+    if (oneOf) {
+      return [
+        ...rules,
+        {
+          oneOf: oneOf.map(oneOfRule => {
```

Comment:
What's going on here? Can you add a comment?

Reply:
It's magic! Will do.